pkg/vpcmodel/jsonOutput.go

```diff
@@ -15,6 +16,20 @@ type connLine struct {
 	Dst                EndpointElem       `json:"dst"`
 	Conn               common.ConnDetails `json:"conn"`
 	UnidirectionalConn common.ConnDetails `json:"unidirectional_conn,omitempty"`
+	connStr            string             // connStr used for sorting
+}
+
+func sortConnLines(connLines []connLine) {
+	sort.Slice(connLines, func(i, j int) bool {
+		if connLines[i].Src.Name() != connLines[j].Src.Name() {
+			return connLines[i].Src.Name() < connLines[j].Src.Name()
+		}
+		if connLines[i].Dst.Name() != connLines[j].Dst.Name() {
+			return connLines[i].Dst.Name() < connLines[j].Dst.Name()
+		}
+
+		return connLines[i].connStr < connLines[j].connStr
```

Comment:
How can we have two lines with the same <src, dst>?
Seems to me like we don't really need `connStr`.

Reply:
Right, fixed.